source/manual/fall-back-to-mirror.html.md

```diff
@@ -73,13 +73,20 @@ Alternatively, we can stop [Nginx](https://www.nginx.com/) on the cache machines
 To stop Nginx on the cache machines, run the following command:
 
 ```bash
-$ fab $environment class:cache incident.fail_to_mirror
+$ gds govuk connect -e production ssh cache:1
```

Comment:
This isn't equivalent - probably needs splitting into the Puppet step and the Nginx step as they both happen in different contexts.

Reply:
I think I'm missing something here - do you mean I need to be clearer about ssh-ing into multiple boxes?
Then... run disable puppet on all first, _then_ disable nginx?

Looking [at this](https://github.com/alphagov/fabric-scripts/blob/master/incident.py#L9-L12), seemed these were run one after another... on the same cash box?

Reply:
Oops, I was half talking rubbish, sorry.

The steps look good, just the intro is confusing: "to stop Nginx, run...ssh" ❌.

Suggest replacing with "SSH onto each cache machine e.g.".

Reply:
gotcha! Yeah that's a good suggestion I'll amend now - thanks 😄!